.github/dependabot.yml

```diff
@@ -72,7 +93,10 @@ updates:
     package-ecosystem: docker
     schedule:
       interval: monthly
-      day: saturday
+    groups:
+      docker-files-raspbian-dependencies:
+        patterns:
+          - "*"
```

Comment:
Please leave the client ones un-grouped. This is: `/docker/docker-files-raspbian`, `/mender-client-acceptance-testing` and `/mender-client-acceptance-testing`. They do not receive updates often and it is not worth the risk.